### ISSUE TITLE
error: [monitor-docker.js]: TypeError: Cannot read property 'id' of undefined

### ISSUE THREAD

**Issue by CoreyLin** [hyperledger/caliper#318]
## Context
The context and env is totally same as #314
In the ending part of the test log, there is an error "error: [monitor-docker.js]: TypeError: Cannot read property 'id' of undefined"
![a2e8739bd69afeb3551893121bb4831](https://user-images.githubusercontent.com/12689769/52894388-9c3b9b00-31e3-11e9-8b2e-7a34299271c8.png)
![2691fcc899c3e2db04a50505e289080](https://user-images.githubusercontent.com/12689769/52894389-9e9df500-31e3-11e9-9368-955fe63dbad1.png)

Could you please check it? Thanks.

**Comment by aklenik** [comment]
Yes, it's been bothering me for some time, it's probably a bug while shutting down the monitoring component. Since it doesn't cause any actual error, it's not a serious bug, but we'll look into it

**Comment by CoreyLin** [comment]
@aklenik OK, thanks. By the way, today is Saturday. You are so hard-working. I didn't expect to get reply today...

**Comment by aklenik** [comment]
Well, didn't have much time recently for the community side of Caliper, so trying to make up for it :) Plus it's in everyone's best interest to make Caliper even more awesome :)

**Comment by CoreyLin** [comment]
Yes, agree. I found it is a very good benchmark tool for hyperledger.
